doc/howtos/building-images.rst

```diff
@@ -230,17 +241,34 @@ The following instructions assume that swupd is not used.
 
 .. TODO: document how to create custom image recipes based on ostro-image.bbclass.
 
+.. _`ostro-image.bbclass`: https://github.com/ostroproject/meta-ostro/blob/master/meta-ostro/classes/ostro-image.bbclass
 
-Installing Additional Packages without swupd
---------------------------------------------
+Installing Additional Packages (without swupd)
+----------------------------------------------
+
+An image derived from ``ostro-image.bbclass`` without additional
+configuration is minimal and establishes a core OS with components
+that must always be present on a device. All additional components beyond
+this minimal configuration must be added explicitly by setting
+``OSTRO_IMAGE_EXTRA_FEATURES`` and/or ``OSTRO_IMAGE_EXTRA_INSTALL`` adding them
+to the image when building without swupd.  
```

Comment:
On Fri, 2016-04-29 at 13:12 -0700, David Kinder wrote:
> @pohly I could use your help rewriting this section then so it's
> technically correct.
> Other changes have been updated and pushed to this PR

Can you work with @igor on this after the release?

Or asking differently, how much of this PR is absolutely required for
the release?